### ISSUE TITLE
Sync current, next staking ledgers to empty ledgers threw

### ISSUE THREAD

**Issue by dannywillems** [MinaProtocol/mina#14044]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/MinaProtocol/mina/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/MinaProtocol/mina/discussions

### Summary

```
File "src/lib/mina_lib/tests/tests.ml", line 492, characters 4-567: Sync current, next staking ledgers to empty ledgers threw
(monitor.ml.Error (Sync_timeout)
  ("<backtrace elided in test>" "Caught by monitor block_on_async")).
  Raised at Base__Exn.protectx in file "src/exn.ml", line 71, characters 4-114
  Called from Ppx_inline_test_lib__Runtime.time_and_reset_random_seeds in file "runtime-lib/runtime.ml", line 356, characters 15-52
  Called from Ppx_inline_test_lib__Runtime.test in file "runtime-lib/runtime.ml", line 444, characters 52-83
  in TEST_MODULE at file "src/lib/mina_lib/tests/tests.ml", line 3, characters 0-22291: Epoch ledger sync tests
  ```

### Steps to replicate this Issue

CI

### Console output

```shell
See https://buildkite.com/o-1-labs-2/mina-o-1-labs/builds/30246#018a6aac-9a8a-4b44-ac4e-21ad8e718dd0/690-1047
```

### Platform

CI

### Build link(s)

https://buildkite.com/o-1-labs-2/mina-o-1-labs/builds/30246#018a6aac-9a8a-4b44-ac4e-21ad8e718dd0/690-1047

**Comment by psteckler** [comment]
`run_test` imposes a timeout of 5 minutes.

Running this test locally: 
```
time ./_build/default/src/lib/mina_lib/tests/.mina_lib_tests.inline-tests/inline_test_runner_mina_lib_tests.exe inline-test-runner mina_lib_tests -only-test tests.ml:493
real	3m20.534s
user	5m7.547s
sys	0m4.689s
```
and the test succeeds.

Is it just a matter of increasing the timeout? There doesn't seem to be a bug.

**Comment by psteckler** [comment]
Finally fixed in #14124.
